src/components/ActivityCollector/attachClickActivityCollector.js

```diff
@@ -24,7 +24,7 @@ const createClickHandler = (eventManager, lifecycle) => {
           return Promise.resolve();
         }
 
-        return eventManager.sendEvent(event);
+        return eventManager.sendEvent(event).then(noop);
```

Comment:
It might be worth adding a comment explaining why we're doing `.then(noop)`. I'm not sure if other developers will understand.

Reply:
yes, you are right. I'll add a comment here and in other places where I have added `then(noop)`.

Reply:
Nice. It may be good to be explicit about what you mean with "we are not returning a value". Specifically, we're trying to avoid returning a value _to the customer_. Consider something like:

```
// eventManager.sendEvent() will return a promise resolved to an
// object and we want to avoid returning any value to the customer
```

Whatever you think would most help other developers.

Reply:
@ninaceban Would you please update the tests accordingly to assert that we're not resolving the promise with a value?